docs/release-notes/release-notes-0.18.0.md

```diff
@@ -120,7 +120,9 @@ call where arguments were swapped.
   node operators to require senders to pay an inbound fee for forwards and
   payments. It is recommended to only use negative fees (an inbound "discount")
   initially to keep the channels open for senders that do not recognize inbound
-  fees. [Send support](https://github.com/lightningnetwork/lnd/pull/6934) is
+  fees. [Positive inbound fees](https://github.com/lightningnetwork/lnd/pull/8627) 
```

Comment:
Nit: Linebreak for every separate PR ?

Reply:
Linebreak yes, but no extra bullet point